src/UserSettingsPage.h

```diff
@@ -175,6 +183,7 @@ class UserSettingsPage : public QWidget
         void moveBack();
         void trayOptionChanged(bool value);
         void themeChanged();
+        void decryptSidebarChanged();
```

Comment:
Could this be put into the UserSettings class, so that you can connect to the settings directly in the timeline manager? Maybe also the theme stuff could be moved and it may clean up some of the signal soup?